python/src/aiconfig/editor/server/server.py

```diff
@@ -182,6 +182,7 @@ def _get_op_args():
 @app.route("/api/add_prompt", methods=["POST"])
 def add_prompt() -> FlaskResponse:
     method_name = MethodName("add_prompt")
+    # TODO: Support specifying index
     signature: dict[str, Type[Any]] = {"prompt_name": str, "prompt_data": Prompt}
```

Comment:
This can be added (maybe follow up diff), because #599 added it to the library.

Aside -- not sure why we've made this so convoluted to call the aiconfig api in the server. Why do we need to do `make_op_run_method` instead of just simply calling `aiconfig.add_prompt`. Looks over-engineered. cc @jonathanlastmileai  @rossdanlm. Not ship-blocking but mentioning this for the near future.

Reply:
Ah, whoops. Not sure how I didn't see it but added now

Reply:
Fair enough, this could be simpler. Basically it reduces boilerplate for input JSON schema checking and constructing HTTP responses.